model/src/main/proto/performance_metrics.proto

```diff
@@ -0,0 +1,149 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Structure for an oppia metric log.
+message OppiaMetricLog {
+  // The timestamp of the event.
+  // unit: long/millis, timezone: UTC
+  int64 timestamp_millis = 1;
+
+  // The priority of the event.
+  Priority priority = 2;
+
+  // Indicates whether the application is in foreground.
+  bool is_app_in_foreground = 3;
+
+  // The memory tier of the current device.
+  MemoryTier memory_tier = 4;
+
+  // The storage tier of the current device.
+  StorageTier storage_tier = 5;
+
+  // The current network type of the device.
+  NetworkType network_type = 6;
+
+  // The current activity where the event is getting logged.
+  CurrentActivity current_activity = 7;
+
+  // Structure of the apk size package.
+  message ApkSize {
+    // Size of the android package file of the app on the device in bytes.
+    int64 apk_size_bytes = 1;
+  }
+
+  // Structure of the storage usage package.
+  message StorageUsage {
+    // Amount of storage usage by the app on user's device in bytes.
```

Comment:
Please document whether this includes the APK size and app cache (since these are normal questions to ask, I think, given the description).

Reply:
done.